.gitignore

```diff
@@ -1,48 +1,49 @@
-
-*.ini
```

Comment:
can you please modify .gitattributes to ensure that this doesn't get changed again in the future?
Maybe we should add a '*' entry?